guides/tools/zoiper-soft-phone.md

```diff
@@ -22,15 +23,34 @@ Soft-phones require basic authentication, so we have to create a user and passwo
   - USERNAME - enter your phone number in E164 format e.g. +441231231231
   - PASSWORD - your password (don't forget to use 1Password)
 - Once it's done click on create. You should see a new record in the CREDENTIAL LISTS field
+- Select newly created credential within SIP Registration Authentication block
 - Save your SIP domain
 
 ## Zoiper Configuration
 
 After installing Zoiper you will be requested to enter the following credentials:
 1. Fill user name (phone number) and password which we created before.
-2. Fill in your hostname, you can find it on [Voice SIP Domains](https://www.twilio.com/console/voice/sip/endpoints) page, e.g. `dvelp-test.sip.twilio.com`
+2. Fill in your localized hostname, you can find it on [Voice SIP Domains](https://www.twilio.com/console/voice/sip/endpoints) page, e.g. `dvelp-test.sip.us1.twilio.com`
 3. Skip Authentication and outbound proxy
 4. Skip testing step
 5. Click Create Account
 
-That's it. Now you are ready to make calls!
+To receive incoming calls use this example of TwiML Bin, to redirect one of your phone numbers to your SIP client
```

Comment:
`In order to receive incoming calls, use this TwiML Bin example to redirect one ...`